superset-frontend/src/components/ReportModal/styles.tsx

```diff
@@ -71,6 +71,7 @@ export const StyledScheduleTitle = styled.div`
 
 export const StyledCronPicker = styled(CronPicker)`
   margin-bottom: ${({ theme }) => theme.gridUnit * 3}px;
+  width: 450px;
```

Comment:
```suggestion
  width: ${({ theme }) => theme.gridUnit * 120}px;
```
Is 480px too much? This seems like a nice round-ish number of gridUnits (divisible by 3 or 4)